### DISCUSSION TITLE
AtGrid 的 iconInfo 如果引用了阿里的iconfont库需要在加上空格否则编译时会用 - 连接

### DISCUSSION THREAD

**Discussion by jackvale** [NervJS/taro#11057] [Q & A]
![image](https://user-images.githubusercontent.com/83709912/148580690-9f4ce1bf-d2e4-4fac-9046-a9fa51331a0b.png)
![image](https://user-images.githubusercontent.com/83709912/148580728-7cd14b34-cd6c-4a2a-9d80-b7ed09485aec.png)
不知道算不算bug希望使用阿里的iconfont注意一下

**Comment by zhushenhang** [comment]
我想知道项目里怎么使用iconfont。你是怎么配置的？

**Comment by yoyo837** [comment]
复制到本地文件夹
